mlops/tests/test_mlops.py

```diff
@@ -294,6 +296,31 @@ def test_general_graph():
     pm.done()
 
 
+def test_mlops_confusion_metrics_apis():
+    pm.init(ctx=None, mlops_mode=MLOpsMode.STAND_ALONE)
+
+    labels_pred = [1, 0, 1, 1, 1, 0]
+    labels_actual = [0, 1, 0, 0, 0, 1]
+    labels_ordered = [0, 1]
+
+    cm = metrics.confusion_matrix(labels_actual, labels_pred, labels=labels_ordered)
+
+    # first way
+    pm.set_stat(ClassificationMetrics.CONFUSION_MATRIX, cm, labels=labels_ordered)
+
+    # second way
+    pm.metrics.confusion_matrix(y_true=labels_actual, y_pred=labels_pred, labels=labels_ordered)
+
+    # should throw error if labels are not provided
+    with pytest.raises(MLOpsException):
+        pm.set_stat(ClassificationMetrics.CONFUSION_MATRIX, cm)
+
+    with pytest.raises(MLOpsException):
+        pm.metrics.confusion_matrix(y_true=labels_actual, y_pred=labels_pred, labels=None)
```

Comment:
Also add checks when the labels_pred and labels_actual are not of the same size.